flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/Environment.java

```diff
@@ -212,6 +226,9 @@ public static Environment merge(Environment env1, Environment env2) {
 		// merge deployment properties
 		mergedEnv.deployment = DeploymentEntry.merge(env1.getDeployment(), env2.getDeployment());
 
+		// merge execution properties
```

Comment:
update comment

Reply:
![got-it](https://github.com/TsReaper/stickers/blob/master/stickers/eastern-timber-wolf/got-it.jpg?raw=true)